### ISSUE TITLE
Sweatpants [BUG]

### ISSUE THREAD

**Issue by Tapionator** [loathers/garbage-collector#1706]
Unfortunately I do not have the text from the error in the GCLI but this happened in both of my runs today. Garbo kept saying it was trying to sweat out booze nonstop and would not resolve until I manually stopped the script and re-ran it.

**Comment by wetflame** [comment]
I am also having this issue. It repeats the following in the gCLI:

```
Executing Postcombat/Fill Sweaty Liver
Autoattack is disabled.

```

Aborting garbo with escape and then re-running seems to work in terms of getting garbo going again.

**Comment by libraryaddict** [comment]
Fixed in #1708
